k8s/provider_test.go

```diff
@@ -514,10 +517,10 @@ func TestShouldCheckMultipleServicesWithGlobalTagsCombinedAndWithPortSpecificTag
 			Tags: []string{"a", "b", "c", "instance:podName_31000"},
 		},
 		{
-			Tags: []string{"a", "b", "c", "secureConnection:true", "instance:podName_31002"},
```

Comment:
You don't have to change this if it fails locally. It doesn't in CI.

Reply:
It was failing without sorting: https://travis-ci.org/github/allegro/consul-registration-hook/builds/727683960

Reply:
Ok, let's do it this way, and get rid of the problem for good:
```go
		tagsSorted := service.Tags
		sort.Strings(tagsSorted)
		assert.Equal(t, expectedServices[i].Tags, tagsSorted)
```